### PR TITLE
Aux Serial config files

### PR DESCRIPTION
**Pull request by ksmola**
The first step in adding Auxilary Serial sensor support. Mainly for Innovate LC-2 wideband controller, at least for now.

### CODE REVIEW

**Comment by rusefi** [comment]
Can one of the admins verify this patch? (meaning the author is not on the white list)

**Comment by rusefi** [comment]
Can one of the admins verify this patch? (meaning the author is not on the white list)

**Comment by rusefi** [comment]
Can one of the admins verify this patch? (meaning the author is not on the white list)

**Comment by rusefi** [comment]
Can one of the admins verify this patch? (meaning the author is not on the white list)

File: firmware/integration/rusefi_config.txt [hunk 1]
```diff
@@ -373,7 +373,7 @@ bit launchControlEnabled;
 bit rollingLaunchEnabled;
 bit antiLagEnabled;
 bit useRunningMathForCranking,Fuel Map,Fixed;
-bit issue_294_25;
+bit displayLogicLevelsInEngineSniffer;
```
Comment (mck1117) [inline firmware/integration/rusefi_config.txt:376]: it looks like you have a merge conflict - can you rebase on master?

Comment (ksmola) [reply firmware/integration/rusefi_config.txt:376]: will do

**Pull request merged**